app/src/main/java/ch/epfl/sdp/blindly/match/Profile.kt

```diff
@@ -0,0 +1,56 @@
+package ch.epfl.sdp.blindly.match
+
+import android.os.Parcel
+import android.os.Parcelable
+
+/**
+ * Class to have the data to display to the matchActivity
+ * We need it to be parcelable to be able to pass them between different activities
+ *
+ * @property id
+ * @property name
+ * @property age
+ */
+data class Profile(
+        val id: Long = counter++,
```

Comment:
I know see why you needed to write `age =` and `name =`. Check if the id is really useful, and if it is, you could convert it as an optional parameter so that you don't have to write.